build error due to default parameter in Markups ROI method

As reported [on Slicer discourse](https://discourse.slicer.org/t/wrong-with-the-module-diffusion/27976/2) the DMRI extension is not available on windows or linux, but it is on mac.

This happened after the [switch to markups](https://github.com/SlicerDMRI/SlicerDMRI/commit/a2ca048a1896df5dcd43ecdffd191297a7ef25ac) and at least on linux this is due to an [`undefined reference to vtkMRMLMarkupsROINode::GetTransformedPlanes(vtkPlanes*, bool)`](https://slicer.cdash.org/viewBuildError.php?buildid=2949139).

--- a/Modules/Loadable/InteractiveSeeding/Logic/CMakeLists.txt
+++ b/Modules/Loadable/InteractiveSeeding/Logic/CMakeLists.txt
@@ -19,7 +19,7 @@ set(${KIT}_TARGET_LIBRARIES
   vtkDMRI
   ${ITK_LIBRARIES}
   vtkSlicerMarkupsModuleMRML
-  vtkSlicerMarkupsModuleMRML
+  vtkSlicerAnnotationsModuleMRML
   vtkSlicerTractographyDisplayModuleMRML
   )
 

--- a/Modules/Loadable/TractographyDisplay/MRML/CMakeLists.txt
+++ b/Modules/Loadable/TractographyDisplay/MRML/CMakeLists.txt
@@ -28,9 +28,9 @@ set(${KIT}_TARGET_LIBRARIES
   vtkDMRI
   ${MRML_LIBRARIES}
   ${VTK_LIBRARIES}
-   vtkSlicerAnnotationsModuleMRML
-   vtkTeem
-   )
+  vtkSlicerMarkupsModuleMRML
+  vtkTeem
+  )
 
 #-----------------------------------------------------------------------------
 SlicerMacroBuildModuleMRML(

